.github/workflows/reusable-tests.yml

```
@@ -190,7 +190,7 @@ jobs:
         '${{ env.TOX_VERSION }}'
 
     - name: Download all the dists
-      uses: actions/download-artifact@v3
+      uses: actions/download-artifact@v4
       with:
         name: ${{ inputs.dists-artifact-name }}
         path: dist/
```

The tests jobs are failing because this is missing:
```suggestion
        merge-multiple: true
        path: dist/
        pattern: ${{ inputs.dists-artifact-name }}*
```